releases/release-4.0.6.md

```diff
@@ -0,0 +1,162 @@
+    - 修复了 logarithm 函数 [#19291](https://github.com/pingcap/tidb/pull/19291)
+    - 修复了当使用 TiDB 落盘时产生的非预期错误 [#19272](https://github.com/pingcap/tidb/pull/19272)
+    - 支持在 index join 的 inner 端使用单个分区表 [#19197](https://github.com/pingcap/tidb/pull/19197)
+    - 修复了对 decimal 产生的错误的 hash 键值 [#19188](https://github.com/pingcap/tidb/pull/19188)
+    - 修复了当了 table endKey 和 region endKey 相同时 TiDB 会产生 no regions 的错误 [#19895](https://github.com/pingcap/tidb/pull/19895)
+    - 修复了 alter partition 的非预期成功 [#19891](https://github.com/pingcap/tidb/pull/19891)
+    - 修复了在下推表达式上，默认最大允许的包长的错误 [#19876](https://github.com/pingcap/tidb/pull/19876)
+    - 修复了在 `ENUM/SET` 列上 `Max/Min` 函数的错误行为 [#19869](https://github.com/pingcap/tidb/pull/19869)
+    - 修复了当 tiflash 节点下线之后，`tiflash_segments` 和 `tiflash_tables` 系统表的读取失败 [#19748](https://github.com/pingcap/tidb/pull/19748)
+    - 修复了 `Count()` 聚集函数的错误结果 [#19628](https://github.com/pingcap/tidb/pull/19628)
+    - 修复了 `TRUNCATE` 操作的运行时错误 [#19445](https://github.com/pingcap/tidb/pull/19445)
+    - 修复了 `PREPARE` statement FROM `@Var` 语句在 `Var` 包含大写字符时候会失败的错误 [#19378](https://github.com/pingcap/tidb/pull/19378)
+    - 修复了在具有大写表名的表上修改 charset 会产生 panic 的错误 [#19302](https://github.com/pingcap/tidb/pull/19302)
+    - 修复了当在包含 `tikv/fiflash` 信息时，`information_schema.statements_summary` 和 `explain` 计划的不一致性 [#19159](https://github.com/pingcap/tidb/pull/19159)
+    - 修复了在测试中 `select into outfile` 出现文件不存在的错误 [#19725](https://github.com/pingcap/tidb/pull/19725)
+
++ PD
+    - 添加 `initial-cluster-token` 配置避免启动时 cluster 之间的通信 [#2922](https://github.com/pingcap/pd/pull/2922)
+    - 修正自动模式下 store limit 的单位 [#2826](https://github.com/pingcap/pd/pull/2826)
+    - 添加对于 scheduler 持久化时引发的错误的处理 [#2818](https://github.com/tikv/pd/pull/2818)
+    - 修复 scheduler 的 http 接口的返回结果可能为空的问题 [#2871](https://github.com/tikv/pd/pull/2871) [#2874](https://github.com/tikv/pd/pull/2874)
+
++ TiKV
+
+    - 修复开启 collation 时对于非 index 列统计信息估算错误的问题 [#8620](https://github.com/tikv/tikv/pull/8620)
+    - 修复当迁移 region 时 green GC 可能错过 lock 的问题 [#8460](https://github.com/tikv/tikv/pull/8460)
+    - 修复 CDC 不合理的 resloved TS 超时等待 [#8573](https://github.com/tikv/tikv/pull/8573)
+    - 修复 TiKV 在极端繁忙下 conf change 可能 panic 的问题 [#8497](https://github.com/tikv/tikv/pull/8497)
+    - 修复 PD client 和其他线程发起 PD sync requests 可能导致死锁的问题 [#8612](https://github.com/tikv/tikv/pull/8612)
+
++ TiFlash
+
+    - 
+
++ Tools
+
+    + BR
+        - 解决数据校验期间可能出现的异常退出的问题 [#479](https://github.com/pingcap/br/pull/479)
+        - 解决 PD leader 切换后可能出现的异常退出的问题 [#496](https://github.com/pingcap/br/pull/496)
+
+    + Dumpling
+        - 解决 binary 类型的 NULL 值没有被正确处理的问题 [#137](https://github.com/pingcap/dumpling/pull/137)
+
+    + TiDB Lightning
+        - 解决 write 和 ingest 失败后依旧显示成功的问题 [#381](https://github.com/pingcap/tidb-lightning/pull/381)
+        - 解决写 checkpoint 不及时的问题 [#386](https://github.com/pingcap/tidb-lightning/pull/386)
```

Comment:
```suggestion
        - 解决写 checkpoint 不及时的问题 [#386](https://github.com/pingcap/tidb-lightning/pull/386)
        
        
## Others

+ TiDB

- 添加对 process list 的 digest [#19829](https://github.com/pingcap/tidb/pull/19829)
- 对于自动提交语句的重试转换到悲观锁模式 [#19796](https://github.com/pingcap/tidb/pull/19796)
- 在 `Str_to_date` 函数中支持 `%r`, `%T` 的数据格式 [#19693](https://github.com/pingcap/tidb/pull/19693)
- 使 `SELECT INTO OUTFILE` 需要文件权限 [#19577](https://github.com/pingcap/tidb/pull/19577)
- 支持 stddev_pop 函数 [#19541](https://github.com/pingcap/tidb/pull/19541)
- 使 DDL 避免长时间重试 [#19488](https://github.com/pingcap/tidb/pull/19488)
- 使 `alter table db.t1 add constraint fk foreign key (c2) references t2(c1)` 语句执行不需要先执行 `use db` [#19471](https://github.com/pingcap/tidb/pull/19471)
- 修复了 `INFORMATION_SCHEMA.CLUSTER_HARDWARE` 不含有 raid 设备信息 [#19457](https://github.com/pingcap/tidb/pull/19457)
- Make dispatch errors be changed from an Error to an Info message in the server log file [#19454](https://github.com/pingcap/tidb/pull/19454)
- 使日志文件中 dispatch errors 从 Error 形式转变为 Info 信息 [#19454](https://github.com/pingcap/tidb/pull/19454)
- 添加 `TiDB-Runtime` dashboard [#19396](https://github.com/pingcap/tidb/pull/19396)
- 使具有 `case-when` 表达式生成列的索引添加操作在遇到 parse 错误时候能够正常退出 [#19395](https://github.com/pingcap/tidb/pull/19395)
- 提升 ALTER TABLE ALGORITHMS 的兼容性 [#19364](https://github.com/pingcap/tidb/pull/19364)
- 在慢日志的计划字段中加入编码好的 insert/delete/update 计划 [#19269](https://github.com/pingcap/tidb/pull/19269)
```

Reply:
Which category does`## others` belong to?

Reply:
I migrate it from English release note  >.<...

Reply:
do you have any idea?

Reply:
Me neither....
@uglyengineer PTAL